src/main/java/org/apache/commons/scxml2/SCXMLConstants.java

```diff
@@ -36,15 +36,6 @@
     public static final String XMLNS_COMMONS_SCXML_PREFIX = "cs";
 
     // W3C SCXML XML Element names
-    public static final String ELEM_ASSIGN = "assign";
```

Comment:
You cannot remove public or protected items as this will break binary compatibility.